plantuml-parser-cli/build/tmp/compileJava/source-classes-mapping.txt

```diff
@@ -0,0 +1,48 @@
+com/shuzijun/plantumlparser/cli/options/fieldmodifier/FieldPrivateModifier.java
```

Comment:
This and a few other files need to be removed.

Reply:
Yes, I added the file path to .gitignore after I pushed. I will delete the files and push again.

Reply:
It's always a pain removing gitignore'd files.